src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (const row of rows) {
+  const copyOfColumn = [];
+
+  copyOfColumn.push(row.cells[1]);
+
+  for (const cell of copyOfColumn) {
+    row.cells[4].insertAdjacentHTML('beforebegin', cell.outerHTML);
+  }
```

Comment:
The loop over `copyOfColumn` is unnecessary because it contains only one element. You can directly insert `row.cells[1].outerHTML` before `row.cells[4]` without using an array.